c/rust-tests/src/tests/blake2b_decl_only.h

```diff
@@ -21,9 +21,25 @@ typedef struct blake2b_state__ {
     uint8_t last_node;
 } blake2b_state;
 
+typedef struct blake2b_param__ {
```

Comment:
If blake2b is used for testing, we can choose: https://crates.io/crates/blake2b-rs

Reply:
Yes, Low-level uses the blake2b-rs.
Now I just wrote a few simple use unit test, I want to wait until the interface is fixed before I complete all of them.